dojo/tools/huskyci/parser.py

```diff
@@ -31,29 +30,30 @@ def parse_json(self, json_output):
         try:
             data = json_output.read()
             try:
-                tree = json.loads(str(data, 'utf-8'))
-            except:
+                tree = json.loads(str(data, "utf-8"))
+            except BaseException:
                 tree = json.loads(data)
-        except:
+        except BaseException:
```

Comment:
```suggestion
            except Exception:
```

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8304/commits/3683282176e99b2e46e9ecd1bd825090dcab1191